docs/modules/ROOT/pages/runbooks/vshn-postgresql.adoc

```diff
@@ -0,0 +1,125 @@
+= PostgreSQL by VSHN
+
+
+[[uptime]]
+== Uptime
+
+[IMPORTANT]
+We don't yet have a lot of operational experience with this service.
+If you received this alert, please add any insights you gained to improve this runbook.
+
+=== icon:glasses[] Overview
+
+The SLI measures the uptime of each PostgreSQL by VSHN instance.
+This SLI is measured by a prober that executes a SQL query every second.
+
+If this SLI results in an alert, it means that a significant number of SQL queries failed and that we risk missing the SLO.
+
+There are two types of alerts that fire if we expect to miss the configured objective.
```

Comment:
why these are called page and ticket alert?

Reply:
Based on https://sre.google/workbook/alerting-on-slos/#6-multiwindow-multi-burn-rate-alerts and Sloths terminology. I.e. "Ticket" alerts should in general result in a ticket or some notification in a dashboard and not directly page someone.